packages/tocco-ui/src/FormattedValue/typeFormatters/SingleSelectFormatter.js

```diff
@@ -10,16 +10,16 @@ const SingleSelectFormatter = ({value, options, breakWords}) => {
   value = js.getOrFirst(value)
   const display = <Typography.Span breakWords={breakWords}>{value.display}</Typography.Span>
 
-  const {tooltips, loadTooltip, DetailLink} = options || {}
+  const {tooltips, loadTooltip, navigationStrategy, linkProps} = options || {}
   const tooltip = _get(tooltips, value.key, null)
 
-  return DetailLink ? (
+  return navigationStrategy?.DetailLink ? (
```

Comment:
Maybe we can put DetailLink into a separate var? And in the end just `return navigationStrategyDetailLink || display` ?